### PR TITLE
new(.gitpod): cloud kernel development environment on gitpod

### PR DESCRIPTION
**Pull request by fntlnz**
Signed-off-by: Lorenzo Fontana <lo@linux.com>

**What type of PR is this?**

/kind documentation

**Any specific area of the project related to this PR?**

/area build

/area driver-kmod

/area driver-ebpf

**What this PR does / why we need it**:

The Falco libraries are used to interact with the Linux kernel via either the Falco [Kernel Module](https://github.com/falcosecurity/libs/tree/master/driver) or the [eBPF probe](https://github.com/falcosecurity/libs/tree/master/driver/bpf).

Contributors might find hard to maintain and use a development environment suitable for Kernel development when they need
to make changes to those components from time to time.

To make that easy, here we have a complete and ready to use environment running as a Gitpod workspace.

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

### CODE REVIEW

File: driver/bpf/Makefile [hunk 1]
```diff
@@ -15,7 +15,7 @@ KERNELDIR ?= /lib/modules/$(shell uname -r)/build
 # DEBUG = -DBPF_DEBUG
 
 all:
-	$(MAKE) -C $(KERNELDIR) M=$$PWD
+	$(MAKE) -n -C $(KERNELDIR) M=$$PWD
```
Comment (fntlnz) [inline driver/bpf/Makefile:18]: I need to revert this, this was not supposed to get in

**Comment by fntlnz** [comment]
Anyone up for a review @leodido @leogr @gnosek ?

**Comment by leogr** [comment]
> Anyone up for a review @leodido @leogr @gnosek ?

I have not yet found the time to give it a try, sorry. 
It's on my list. Hoping I will take a look this week. :smiley_cat:

**Review by leogr — COMMENTED** [review]
The feature seems to be very cool :star_struck: 

Anyway, I have tried to test this by running
https://gitpod.io/#https://github.com/fntlnz/libs/tree/cloud-kernel-development-environment

The provided VM built (it took a while), then I followed the instructions.
Eventually, I was not able to build the bpf probe :( (no errors, just the file is missing, see the screenshot)
Screenshot from 2021-06-11 15-10-19

At the moment I'm not sure what the problem could be :thinking: 
Any idea?

**Pull request closed without merging**